packages/ses/NEWS.md

```diff
@@ -31,8 +31,10 @@ User-visible changes in SES:
   under SES to specifically allow the use of `something.import()` or
   `something.eval()` methods.
 - Fix: `new Compartment(null, null, options)` no longer throws.
+- New lockdown option: `overrideDebug: [...props]` to detect where a given
+  override enablement is required.  Most useful as `overrideTaming: 'severe', overrideDebug: ['constructor']`.
```

Comment:
Your text suggests that the best course of action is to add a new enablement. Better to suggest they fix the assignment in question. Something like

```suggestion
- New lockdown option: `overrideDebug: [...props]` to detect where a property assignment needs to be turned into a `defineProperty` to avoid the override mistake.  Most useful as `overrideTaming: 'severe', overrideDebug: ['constructor']`.
```

Reply:
Thanks, that's much clearer!